epub33/core/index.html

```diff
@@ -282,6 +282,23 @@ <h3>Terminology</h3>
 							media types designed for optimum compression or that provide optimized streaming
 							capabilities.</p>
 					</dd>
+
+					<dt>
+						<dfn id="dfn-container-root-url">Container Root URL</dfn>
+					</dt>
+					<dd>
+						<p>The (implementation specific) <a data-cite="url#concept-url">URL</a> representing the <a>Root Directory</a> of an <a>OCF Abstract Container</a>, defined in <a href="#sec-container-iri"></a>.</p>
```

Comment:
The use of parentheses here is ambiguous. Is it saying it is always implementation-specific or that it could be?

I'd suggest removing the parentheses and explaining what is meant here in a second sentence.

Reply:
Right, the text has been improved in the definition itself but has not been retrofitted to the terminology section. I have changed it and now it says:

> The [URL](https://url.spec.whatwg.org/#concept-url) [[URL]] of the [Root Directory](#dfn-root-directory) representing the [OCF Abstract Container](#dfn-ocf-abstract-container). It is implementation specific, but EPUB Creators MUST assume it has properties defined in [§ 6.1.5 URLs in the OCF Abstract Container](#sec-container-iri").

Does that work?

Reply:
Lowercase the must, as we shouldn't have normative requirements in terminology, and it works for me.

Reply:
Oops, my bad. Changed.